### DISCUSSION TITLE
Offline installation of docsets

### DISCUSSION THREAD

**Discussion by AHeinlein** [zealdocs/zeal#1540] [Q&A] [answered]
Hello,
I just came across Zeal which looks quite nice for my purposes. However, I am looking for a way to install docsets offline - I want to use zeal on machines that never have Internet connection. Is that possible, and if so, how?
Thank you!

**Comment by trollixx** [comment, marked as answer]
There's no built-in functionality for this, but you can copy docset storage folder between machines.
